src/current/v25.2/architecture/transaction-layer.md

```diff
@@ -432,6 +432,45 @@ Despite their logical equivalence, the transaction coordinator now works as quic
 
 Additionally, when other transactions encounter a transaction in `STAGING` state, they check whether the staging transaction is still in progress by verifying that the transaction coordinator is still heartbeating that staging transaction's record. If the coordinator is still heartbeating the record, the other transactions will wait, on the theory that letting the coordinator update the transaction record with the result of the attempt to commit will be faster than going through the transaction status recovery process. This means that in practice, the transaction status recovery process is only used if the transaction coordinator dies due to an untimely crash.
 
+### Buffered writes
+
+{% include_cached new-in.html version="v25.2" %} Buffered Writes enhance transaction throughput and reduce operational cost by minimizing the number of round-trips between the client and the database during write operations.
+
+{% include feature-phases/preview.md %}
+
+Buffered Writes work by temporarily storing a transaction's writes on the client-side ([gateway node]({% link {{ page.version.version }}/architecture/sql-layer.md %}#gateway-node)) until the transaction [commits](#parallel-commits). This approach reduces redundant writes, minimizes [pipeline](#transaction-pipelining) stalls, and allows for passive use of the [1-phase commit (1PC) fast-path]({% link {{ page.version.version }}/ui-distributed-dashboard.md %}#kv-transactions). By deferring writes until commit time, the system can serve read-your-writes locally, resulting in further potential performance gains.
```

Comment:
I think ending with write-your-writes locally maybe gives it too much prominence. For most users, the ability to potentially get 1PC for explicit transactions is likely the biggest win.

Reply:
revised to move read-your-writes earlier in the list, and end with 1PC & call out its potential performance gains explicitly:

> Buffered Writes work by temporarily storing a transaction's writes on the [gateway node]({% link {{ page.version.version }}/architecture/sql-layer.md %}#gateway-node) until the transaction [commits](#parallel-commits). This approach reduces redundant writes, minimizes [pipeline](#transaction-pipelining) stalls, and allows the system to serve read-your-writes locally. Most importantly, it allows for passive use of the [1-phase commit (1PC) fast-path]({% link {{ page.version.version }}/ui-distributed-dashboard.md %}#kv-transactions) which can lead to increased performance.